docs/core/install/includes/package-manager-switcher.md

```diff
@@ -14,3 +14,5 @@
 > - [OpenSUSE 15 - x64](../linux-package-manager-opensuse15.md)
 > - [SLES 12 - x64](../linux-package-manager-sles12.md)
 > - [SLES 15 - x64](../linux-package-manager-sles15.md)
+
+_Package manager installs are only supported on **64-bit** CPUs_. For more information, see [.NET Core dependencies and requirements](../dependencies.md), [Manually install the .NET Core SDK](../sdk.md?pivots=os-linux#download-and-manually-install), and [Manually install the .NET Core Runtime](../runtime.md?pivots=os-linux#download-and-manually-install).
```

Comment:
To me, "more information" would be about *why* it only supports x64--it seems like this should have a positive action rather than a handful of links to choose from. More along the lines of

> _Package manager installs are only supported on **64-bit** CPUs_. On other devices, use the instructions to [Manually install the .NET Core SDK](../sdk.md?pivots=os-linux#download-and-manually-install) or [Manually install the .NET Core Runtime](../runtime.md?pivots=os-linux#download-and-manually-install). For more information about supported platforms, see [.NET Core dependencies and requirements](../dependencies.md), .

The "install .NET Core" button on dot.net drops you here without having clicked anything about arch, so I think we need to make it very easy to get pointed in the right direction after the detour.